package.json

```diff
@@ -5,6 +5,7 @@
   "license": "MIT",
   "private": true,
   "dependencies": {
+    "aws-cdk": "^2.76.0",
```

Comment:
?

Reply:
I found this needed to be the same version as in requirements-dev.txt or I get an error about mismatching versions. But perhaps this is only if the virtualenv is not setup correctly, I will check.

Reply:
interesting! 

the `cdk` depends on `aws-cdk` so my understanding is that when installing `cdk` it should install the right `aws-cdk` (https://www.npmjs.com/package/cdk?activeTab=code)

Reply:
removed